pkg/deploy/util/util.go

```diff
@@ -416,3 +444,16 @@ func (s ByMostRecent) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s ByMostRecent) Less(i, j int) bool {
 	return !s[i].CreationTimestamp.Before(s[j].CreationTimestamp)
 }
+
+// ControllersByCreationTimestampAsc sorts a list of ReplicationControllers by creation timestamp oldest first, using their names as a tie breaker.
+type ControllersByCreationTimestampAsc []api.ReplicationController
```

Comment:
Those use deployment version, not timestamp, and I wanted to keep this as a clean port of upstream logic as possible. Can switch if that's better.